Modules/TPC/src/QualityObserver.cxx

```diff
@@ -90,8 +109,6 @@ void QualityObserver::update(Trigger t, framework::ServiceRegistryRef services)
 void QualityObserver::finalize(Trigger t, framework::ServiceRegistryRef)
 {
   generatePanel();
-  delete mCanvas;
-  mCanvas = nullptr;
```

Comment:
This is indeed correct to be removed, but please delete `mCanvas` in `~QualityObserver` instead.

Reply:
Putting it into  `~QualityObserver` somehow then still had some issues for me. That's why I just removed this lines and treated `mCanvas` similar to `mPlots` in the `Framework/TrendingTask`

Reply:
I see, thanks for the explanation. To be understood.